pontoon/sync/vcs/models.py

```diff
@@ -37,6 +40,37 @@
 log = logging.getLogger(__name__)
 
 
+class DownloadTOMLParser(TOMLParser):
+    """
+    This wrapper is a workaround for the lack of the shared and persistent filesystem
+    on Heroku workers.
+    Related: https://bugzilla.mozilla.org/show_bug.cgi?id=1530988
+    """
+
+    def __init__(self, checkout_path, permalink_prefix):
+        self.checkout_path = checkout_path
+        self.permalink_prefix = permalink_prefix
+
+    def parse(self, path, env=None, ignore_missing_includes=True):
+        config_path = path.replace(self.checkout_path, "")
```

Comment:
Do we need that `replace()`?

IIUC, the `path` is `self.configuration_file`, which doesn't include the `self.checkout_path`.

Reply:
Unfortunately, that's not so simple :/ When TOMLParser parses the main project file and it contains in the project config files, `parse()` is called again for every referenced file and passes its absolute path.